Sources/Event Logging/TracksService.m

```diff
@@ -229,7 +230,8 @@ - (void)switchToAuthenticatedUserWithUsername:(NSString *)username userID:(NSStr
     self.token = token;
 
     #if TARGET_OS_IPHONE
-    [ExPlat configureWithPlatform:_eventNamePrefix oAuthToken:token userAgent:self.userAgent anonId:anonymousID];
+    NSString *platform = _platform ? _platform : _eventNamePrefix;
+    [ExPlat configureWithPlatform:platform oAuthToken:token userAgent:self.userAgent anonId:anonymousID];
```

Comment:
To avoid breaking other repositories, this property be left `nil`, and `eventNamePrefix` will be used instead.